### ISSUE TITLE
Liquid tag 'uri'  is not working correctly

### ISSUE THREAD

**Issue by jsmadis** [3scale/apicast#890]
I am using URL rewriting policy with 'value' evaluated as liquid. The value is not returned correctly. It works with liquid `{{ service.id }}`, or `{{ host }}`, but not with `{{ uri }}`.
 
**policy chain:**
```
"policy_chain": [
      {
        "name": "apicast",
        "version": "builtin",
        "configuration": {
          
        }
      },
      {
        "name": "url_rewriting",
        "version": "builtin",
        "configuration": {
          "query_args_commands": [
            {
              "op": "set",
              "arg": "arg",
              "value_type": "liquid",
              "value": "{{ uri }}"
            }
          ]
        }
      }
    ],
```
**request response:**

```
{
  "args": {
    "arg": "", 
    "user_key": "rate-limit-app-1-jsmadis-wjodncie"
  }, 
  "headers": {
    "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8", 
    "Accept-Encoding": "gzip, deflate, br", 
    "Accept-Language": "en-US,en;q=0.9", 
    "Connection": "close", 
    "Cookie": "pl_authenticated=..."
    "Forwarded": "for=10.40.2.111;host=rate-limit-api-jsmadis-obwvarye-3scale-staging.<3scale_url>;proto=https", 
    "Host": "httpbin.org", 
    "Upgrade-Insecure-Requests": "1", 
    "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.81 Safari/537.36", 
    "X-3Scale-Proxy-Secret-Token": "Shared_secret_sent_from_proxy_to_API_backend_78a28d5a37aa56ca", 
    "X-Forwarded-Host": "rate-limit-api-jsmadis-obwvarye-3scale-staging.<3scale_url>"
  }, 
  "origin": "10.40.2.111, 66.187.233.202", 
  "url": "https://rate-limit-api-jsmadis-obwvarye-3scale-staging.<3scale_url>/get?user_key=rate-limit-app-1-jsmadis-wjodncie&arg="
}


```

**Comment by davidor** [comment]
I can confirm this is a bug. 

The problem is that when rendering the template strings, the URL rewriting policy uses the "policies context" without merging it with what the `ngx_variable` module exposes. This is a bug that might happen with other policies too. I think that merging the policies context with the `ngx_variable` one should be done in the `TemplateString` module to avoid doing it in every policy.

**Comment by davidor** [comment]
Fixed in #891
